middleware.ts

```diff
@@ -13,7 +13,8 @@ const pathsToMatch = [
   "/feed/",
   "/user/notifications",
   "/user/settings",
-  "/account-deleted"
+  "/account-deleted",
+  "/workspaces/:path*",
```

Comment:
I'm curious how demo insights and demo lists play out in the context of workspaces @bdougie @isabensusan

Reply:
I'd lean towards showing demo or default workspace, i.e. topJS projects or am organization like vuejs as the default.

Reply:
Sounds good. It won't be part of this PR, but will probably come closer to the end of this whole feature.